sources/main.cpp

```diff
@@ -94,8 +94,8 @@ int main(int argc, char* argv[]) {
     bool configError = false;
     if (!QFile::exists(cmdLineHandler.configFile())) {
         qFatal("App configuration file not found: %s", cmdLineHandler.configFile().toLatin1().constData());
-        return 1;
```

Comment:
I'd leave the return value to make it crystal clear that it's game over. Not everyone knows that qFatal terminates the process.

Reply:
I removed because Qt is complaining that it will not reach that.